src/__internal__/focus-trap/focus-trap.test.tsx

```diff
@@ -238,723 +178,727 @@ describe("FocusTrap", () => {
+        expect(screen.getByRole("dialog")).toHaveFocus();
       });
     });
 
     describe("and some children elements are disabled", () => {
-      beforeEach(() => {
-        render(
+      it("only focuses those that are not", async () => {
+        const { user } = setup(
           <MockComponent>
-            <button type="button">{BUTTON_ONE}</button>
+            <button type="button">One</button>
             <button type="button" disabled>
               Disabled button One
             </button>
-            <button type="button">{BUTTON_TWO}</button>
+            <button type="button">Two</button>
             <button type="button" disabled>
               Disabled button two
             </button>
           </MockComponent>
         );
-      });
+        const buttonOne = screen.getByRole("button", { name: "One" });
+        buttonOne.focus();
 
-      it("only focuses those that are not", async () => {
-        focusElement(screen.getByText(BUTTON_ONE));
-        await tabPress();
-        expect(screen.getByText(BUTTON_TWO)).toHaveFocus();
-        await tabPress();
-        expect(screen.getByText(BUTTON_ONE)).toHaveFocus();
+        await user.tab();
+        const buttonTwo = screen.getByRole("button", { name: "Two" });
+        expect(buttonTwo).toHaveFocus();
+
+        await user.tab();
+        expect(buttonOne).toHaveFocus();
       });
     });
   });
 
   describe("when first focusable elements are radio buttons", () => {
-    beforeEach(() => {
-      render(
-        <MockComponent>
-          <RadioButtonGroup
-            name="mybuttongroup"
-            legend="How do you want to create this address?"
-            legendInline
-            onChange={() => jest.fn()}
-            value="1"
-            legendWidth={40}
-          >
-            <RadioButton value="1" label={RADIO_LABEL_ONE} size="large" />
-            <RadioButton value="2" label={RADIO_LABEL_TWO} size="large" />
-          </RadioButtonGroup>
-          <button type="button">{BUTTON_ONE}</button>
-          <button type="button">{BUTTON_TWO}</button>
-        </MockComponent>
-      );
-    });
+    const WithRadioGroup = () => (
+      <MockComponent>
+        <RadioButtonGroup name="Colours" legend="Colours" onChange={() => {}}>
+          <RadioButton value="Red" label="Red" />
+          <RadioButton value="Green" label="Green" />
+        </RadioButtonGroup>
+        <button type="button">One</button>
+        <button type="button">Two</button>
+      </MockComponent>
+    );
 
-    describe("when focus on first radio button shift-tab pressed", () => {
-      it("should loop focus to the last focusable element", async () => {
-        focusElement(screen.getByLabelText(RADIO_LABEL_ONE));
-        await shiftTabPress();
-        expect(screen.getByText(BUTTON_TWO)).toHaveFocus();
-      });
+    it("loops focus to the last focusable element, when focus on first radio button shift-tab pressed", async () => {
```

Comment:
praise: much improved language/description of the test

Applies to all of those changes